project_tests.sh

```diff
@@ -2,5 +2,17 @@
 
 set -e
 
-pytest sciencebeam_utils && \
-    pylint sciencebeam_utils
+# avoid issues with .pyc/pyo files when mounting source directory
+export PYTHONOPTIMIZE=
+
+echo "running tests"
+pytest -p no:cacheprovider
+
+echo "running pylint"
+PYLINTHOME=/tmp/sciencebeam-utils \
```

Comment:
why is this necessary?

Reply:
To be honest it won't be necessary in this case. It was in another project where the Docker user wasn't root. Seems to make sense not to write to the current directory but I could reove that.

Reply:
It looks like one of those things that either is removed now or will stay there forever; as you like

Reply:
I think I will keep it in. As it would could also cause issues when mounting the source directory and is consistent with other projects. I want `pylint` to only report and don't need output files in the project folder.